mojo-executor/src/main/java/org/twdata/maven/mojoexecutor/MojoExecutor.java

```diff
@@ -104,13 +113,26 @@ public static void executeMojo(Plugin plugin, String goal, Xpp3Dom configuration
             MojoDescriptor mojoDescriptor = pluginDescriptor.getMojo(goal);
             if (mojoDescriptor == null) {
                 throw new MojoExecutionException("Could not find goal '" + goal + "' in plugin "
-                        + plugin.getGroupId() + ":"
-                        + plugin.getArtifactId() + ":"
-                        + plugin.getVersion());
+                    + plugin.getGroupId() + ":"
+                    + plugin.getArtifactId() + ":"
+                    + plugin.getVersion());
             }
             MojoExecution exec = mojoExecution(mojoDescriptor, executionId, configuration);
             env.getPluginManager().executeMojo(session, exec);
-        } catch (Exception e) {
+        // TODO : Consider using JDK7 multi-catch
+        } catch (PluginConfigurationException e) {
```

Comment:
Yeah, I think it's probably OK to drop EOLed Java versions (with a major version bump).